services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
```

Comment:
Consider using `Optional[str]` from `typing` for `title` to explicitly indicate that this parameter can be None.